src/main/java/org/apache/sling/api/wrappers/package-info.java

```diff
@@ -17,7 +17,7 @@
  * under the License.
  */
 
-@Version("2.6.2")
+@Version("3.0.0")
```

Comment:
This is the recommendation from the tooling. It has to do with the default methods that the tooling doesn't take into account. To implement the default method in this class, I removed the methods that were there. The tooling see's that as a change in the API.

Reply:
Can you add the output here? I think we should not change API in a breaking way, especially in the API bundle.

Reply:
There is no breaking change, the analysis just doesn't take default methods into account. However to prevent a major bump I've copied the default methods into the CompositeValueMap so that it is changed into a minor bump of 2.6.3